### PR TITLE
Add all doors and fix system

### PR DESCRIPTION
**Pull request by torresmon**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
add exceptions for odd door ids

**Issues addressed:** 

fix odd id doors

[cont]: https://github.com/EPuncker/orts2/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: data/scripts/actions/others/doors.lua [hunk 1]
```diff
@@ -1,3 +1,20 @@
+local openoddDoors = {
```
Comment (EPuncker) [inline data/scripts/actions/others/doors.lua:1]:
```suggestion
local openOddDoors = {
```

File: data/scripts/actions/others/doors.lua [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+	[12695] = { locked = 12692, closed = 12692 },
+	[12703] = { locked = 12701, closed = 12701 },
+	[33432] = { locked = 33429, closed = 33428 },
+	[33433] = { locked = 33431, closed = 33430 }
+}
+local closedoddDoors = {
```
Comment (EPuncker) [inline data/scripts/actions/others/doors.lua:7]:
```suggestion
local closedOddDoors = {	
```

File: data/scripts/actions/others/doors.lua [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+	[33432] = { locked = 33429, closed = 33428 },
+	[33433] = { locked = 33431, closed = 33430 }
+}
+local closedoddDoors = {	
+	[12692] = { locked = 12692, open = 12695 },
+	[12701] = { locked = 12701, open = 12703 },
+	[33428] = { locked = 33429, open = 33432 },
+	[33430] = { locked = 33431, open = 33433 }
+}
+local lockedoddDoors = {
```
Comment (EPuncker) [inline data/scripts/actions/others/doors.lua:13]:
```suggestion
local lockedOddDoors = {
```

**Comment by torresmon** [comment]
mais tarde vou testar tudo com calma e se não encontrar erros faço o pr no tfs em seguida

**Review by EPuncker — APPROVED** [review]

**Pull request merged**